.github/workflows/publish-prisma-fmt-wasm.yml

```diff
@@ -0,0 +1,68 @@
+name: Build and publish @prisma/prisma-fmt-wasm
+
+concurrency: build-prisma-fmt-wasm
+
+on:
+  workflow_dispatch:
+    inputs:
+      enginesWrapperVersion:
+        required: true
+      enginesHash:
+        required: true
+      npmDistTag:
+        required: true
+        default: 'latest'
+
+jobs:
+  build:
+    name: Build and publish @prisma/prisma-fmt-wasm
+    runs-on: ubuntu-latest
+    steps:
+      - name: Print input
+        env:
+          THE_INPUT: '${{ toJson(github.event.inputs) }}'
+        run: |
+          echo $THE_INPUT
+
+      - uses: actions/checkout@v3
+        ref: ${{ github.event.inputs.enginesHash }}
+      - uses: cachix/install-nix-action@v16
+
+      #
+      # Build
+      #
+
+      - run: nix build .#prisma-fmt-wasm
+
+      #
+      # Publish
+      #
+
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '14.x'
+
+      - name: Set up NPM token
+        run: echo "//registry.npmjs.org/:_authToken=${{ secrets.NPM_TOKEN }}" > ~/.npmrc
+
+      - run: |
+          PACKAGE_DIR=$( nix run .#renderPrismaFmtWasmPackage ${{ github.event.inputs.enginesWrapperVersion }})
+          npm publish "$PACKAGE_DIR" --access public --tag ${{ github.event.inputs.npmDistTag }}
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
```

Comment:
Note: Let's check that `NPM_TOKEN` is set in the settings (needs admin rights though)

Reply:
Having a NPM_TOKEN in this repo is quite a change. As those are not scoped in any way afaik, this feels a bit scary. @Jolg42? (Can we scope them? Any alterantives?)

Reply:
Alternatively, we have the full package in this CI step. We could put it in a tarball and upload it somewhere else (maybe github releases), and have something else then publish it to npm.

Reply:
> As those are not scoped in any way afaik

does not make a difference here, but out of curiosity: are github secrets not scoped (e.g. to a specific workflow), or are npm tokens not scoped (e.g. to a specific package)?